<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMenu</class>
 <widget class="QMainWindow" name="AdminMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 235, 235);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Login_view">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>701</width>
      <height>481</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 127);
color: qlineargradient(spread:reflect, x1:1, y1:1, x2:1, y2:0.494, stop:0 rgba(54, 54, 54, 199), stop:1 rgba(255, 255, 255, 255));
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>361</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>ADMIN MENU</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>561</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="LihatInvestaris">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(170, 255, 255);
border-radius : 10px;</string>
      </property>
      <property name="text">
       <string>Lihat Inventaris</string>
      </property>
     </widget>
     <widget class="QPushButton" name="KelolaAkun">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(170, 255, 255);
border-radius : 10px;</string>
      </property>
      <property name="text">
       <string>Kelola Akun</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Peminjaman">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(170, 255, 255);
border-radius : 10px;</string>
      </property>
      <property name="text">
       <string>Peminjaman</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Pengembalian">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(170, 255, 255);
border-radius : 10px;</string>
      </property>
      <property name="text">
       <string>Pengembalian</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TerimaInventaris">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(170, 255, 255);
border-radius : 10px;</string>
      </property>
      <property name="text">
       <string>Terima Inventaris</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icon/LogOut.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="LogOutButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
